extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -140,8 +140,64 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		await this.bookTocManager.createBook(bookPath, contentPath);
 	}
 
-	async editBook(book: BookTreeItem, section: BookTreeItem): Promise<void> {
-		await this.bookTocManager.updateBook(section, book);
+	async editBook(movingElement: BookTreeItem): Promise<void> {
+		try {
+			let bookOptions: vscode.QuickPickItem[] = [];
+			let pickedSection: vscode.QuickPickItem;
+			let updateBook: BookTreeItem;
+			let bookSections: JupyterBookSection[];
+			this.books.forEach(book => {
+				if (!book.isNotebook) {
+					bookOptions.push({ label: book.bookItems[0].title, detail: book.bookPath });
+				}
+			});
+			let pickedBook = await vscode.window.showQuickPick(bookOptions, {
+				canPickMany: false,
+				placeHolder: loc.labelBookFolder
+			});
+
+			if (pickedBook && movingElement) {
+				updateBook = this.books.filter(book => book.bookPath === pickedBook.detail)[0].bookItems[0];
```

Comment:
books.find() might be better here, since you always take the first element that matches.